source/frontend/src/features/mapSideBar/property/MotiInventoryContainer.tsx

```diff
@@ -120,10 +138,8 @@ export const MotiInventoryContainer: React.FunctionComponent<
       onClose={props.onClose}
     >
       <>
-        <PropertyViewSelector
+        <PropertyRouter
           composedPropertyState={composedPropertyState}
```

Comment:
this composed property state might not be necessary anymore if the router can handle the expected state

Reply:
composedPropertyState is maybe a misnomer. This is the object that contains all of the ltsa/property/bc assessment information, as well as the loading states for each. Still required.